ddtrace/appsec/_iast/_taint_tracking/aspects/aspect_format.cpp

```diff
@@ -17,13 +18,14 @@ api_format_aspect(StrType& candidate_text,
                   const py::args& args,
                   const py::kwargs& kwargs)
 {
-    const auto tx_map = Initializer::get_tainting_map();
+    auto tx_map = taint_engine_context->get_tainted_object_map_from_list_of_pyobjects(
```

Comment:
very small nit/suggestion/syntactic sugar (feel free to ignore): since this is called a lot and it's pretty long and `taint_engine_context` is basically global, you could add a shorter global helper in `taint_engine_context.h` like:

`auto tx_map = map_from_pyobjects({...})`

Reply:
do you mean, create a helper like...
```
TaintedObjectMapTypePtr
map_from_pyobjects(param){
return taint_engine_context->get_tainted_object_map_from_list_of_pyobjects(param);
}
```
?

Reply:
Yes. Just syntactic sugar for this expression since it's called very frequently.